.github/workflows/asterion-aws-pr.yml

```diff
@@ -0,0 +1,42 @@
+name: asterion-aws-infra-ci-pr
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  preview:
+    defaults:
+      run:
+        shell: bash
+        working-directory: infra-aws
+    name: Preview AWS Infrastructure
+    runs-on: ubuntu-latest
+    steps:
+      - name: Import Github actions
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
We'll need to populate these in github: https://github.com/asterion-digital/asterion-as-code/settings/secrets/actions

Do we have an organisation level account set up for asterion?

Reply:
Not yet - I'll need to create the organization and update users, account billing, etc to point to Asterion assets.